src/routes/data-decode/entities/data-decoded-parameter.entity.ts

```diff
@@ -8,13 +8,19 @@ export class DataDecodedParameter {
   @ApiProperty()
   value: unknown;
   @ApiProperty()
-  valueDecoded?: Record<string, any> | Record<string, any>[] | undefined;
+  valueDecoded?:
+    | Record<string, unknown>
+    | Record<string, unknown>[]
```

Comment:
OOC how are these serialised in the Swagger interface?

Reply:
Both before and after the changes in this PR, it is shown as `{}`.